cluster/sbin/snappy-nodes.sh

```diff
@@ -243,6 +243,14 @@ function execute() {
       -*) postArgs="$postArgs $arg"
     esac
   done
+  #copy the conf files into other node before starting the launch processs
+  if [ -n "$isStart" ]; then
+   #check $host is present in memberArr or not. If not then copy else do not copy
+    if [[ ! " ${memberArray[@]} " =~ " ${host} " ]]; then
```

Comment:
I think we can remove memberArray and instead rely on the file diff you do before copy, to avoid copying the file multiple times.

Reply:
Removed memberArray